random-beans/src/main/java/io/github/benas/randombeans/randomizers/misc/EnumRandomizer.java

```diff
@@ -32,7 +32,7 @@
  *
  * @author Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
  */
-public class EnumRandomizer<T extends Enum<T>> extends AbstractRandomizer<Enum<T>> {
+public class EnumRandomizer<T> extends AbstractRandomizer<T> {
```

Comment:
Hi,

I completely agree.

I changed the old generic signature because it was "wrong"/ugly to use. The new one was less verbose for the user (see other changes of the same commit), but as you said it was too permissive.

I have update the signature again. The new version combines the best of both versions. It's nice to use, but requires the parameter to be a `Enum` with the right type.

Thanks for the review. :)

Cheers,
Pascal